srcpkgs/pywatchman/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'pywatchman'
+pkgname=pywatchman
+version=1.4.1
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="BSD"
+homepage="https://facebook.github.io/watchman/"
+distfiles="${PYPI_SITE}/p/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=d0047eb275deafb0011eda0a1a815fbd9742478c3d2b5ad6956d300e447dc2f9
+alternatives="watchman-make:watchman-make:/usr/bin/watchman-make2 watchman-wait:watchman-wait:/usr/bin/watchman-wait2"
+
+post_install() {
+	vlicense PKG-INFO
```

Comment:
the package from https://pypi.org/project/pywatchman/ does not have LICENSE file

Reply:
![pywatch](https://user-images.githubusercontent.com/28383668/48670914-38ff7700-ead5-11e8-935c-eb032ca6fff3.png)

Reply:
Just put it in the distfiles then

vlicense ../LICENSE.txt